src/components/search/popular-results/PopularResults.jsx

```diff
@@ -27,20 +30,30 @@ const PopularResults = ({
         )}
       </h2>
       {isSearchStalled && (
-        <div className="row">
-          {[...Array(numberResultsToDisplay).keys()].map(resultNum => (
-            <div key={resultNum} className="skeleton-course-card">
-              {getSkeletonCardFromTitle(title)}
-            </div>
-          ))}
-        </div>
+        <CardGrid
```

Comment:
Could be extracted into its own utility component if we plan to reuse this pattern throughout the MFE.
I ended up extracting it in the admin portal. I don't know if its a pattern we want to bring out as a utility component we want to use in other MFEs after highlights are complete.

See https://github.com/openedx/frontend-app-admin-portal/pull/912/files#diff-e1cade14f1ff31cdabee8d60f4fa869954dbfd4b719ad49933b1f4c40f42e936

Reply:
Good suggestion; I abstracted the `columnSizes` prop to a `CARDGRID_COLUMN_SIZES` constant for consistency in card grid layout within the `SearchResults` component.